source/ed/ed2nav.cpp

```diff
@@ -383,7 +383,7 @@ int ed2nav(int argc, const char* argv[]) {
 
     read = (pt::microsec_clock::local_time() - start).total_milliseconds();
```

Comment:
It would probably be easier and more consistent to use `universal_time()` everywhere all along that file.